client/src/redux/reducers/theme.ts

```diff
@@ -0,0 +1,24 @@
+import { ActionType, Action } from '../actionTypes/theme';
+
+interface ThemeState {
+  theme: string;
+}
+
+const initialState = {
+  theme: 'light'
```

Comment:
I think in this place you can write `initialState = 'light'`
Then you can use in app `state.theme`, but not `state.theme.theme` like in Header and Input (maybe in other components)

Reply:
How do you propose to update it then in themeReducer function?

Reply:
Maybe return `return action.payload`.
I don't know. Maybe you are right

Reply:
agree with Andrew comment, `state.theme` can be a simple string